src/util.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+const axios = require("axios");
```

Comment:
Looks like you need to run prettier again? I'm seeing a lot of double-quotes.

Reply:
Ah, turns out my prettier file was magically empty again.

Reply:
It should contain the following:

```
---
bracketSpacing: false
printWidth: 80
semi: true
singleQuote: true
tabWidth: 2
trailingComma: es5
useTabs: false
```